docs/content/en/docs/Coding Milestones/PoC2023q1/environments/dev-env.md

```diff
@@ -6,6 +6,479 @@ description: >-
  
 ---
 
-## Heading
+##Get Started Using the KCP-Edge Development Environment.
 
-under construction - coming soon
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/README.md)
+
+#### 1. Create some edge infrastructure (edge pclusters):
+
+  - For example: create a kind cluster
+
+    ```bash
+    kind create cluster --name florin
+    ``` 
+    
+#### 2. Deploy the KCP-Edge platform:
+    
+  * Step-1: Clone this repo:
+
+    ```bash
+    git clone -b dev-env-v2 https://github.com/dumb0002/edge-mc.git
+    ```
+
+  * Step-2: change into the following directory path:
+
+    ```bash
+      cd edge-mc/environments/dev-env
+    ```
+
+  * Step-3: Deploy kcp-edge
+
+    ```bash
+    ./install_edge-mc.sh --stage 0
+    ```
+
+    This will start `kcp` and create/deploy the following components:
+
+    - 4 kcp workspaces: edge service provider workspace (`espw`), inventory management workspace (`imw`) and workload management workspace (`wmw`) under my-org workspace
+
+    ```bash
+    kubectl ws tree
+    .
+    └── root
+        ├── compute
+        ├── espw
+        ├── imw-1
+        └── my-org
+            └── wmw-1
+    ```
+    - 3 kcp-edge controllers: [edge-scheduler](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/edge-scheduler/), [mailbox-controller](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/mailbox-controller/) and [placement-translator](https://docs.kcp-edge.io/docs/coding-milestones/poc2023q1/placement-translator/)
+
+    ```bash
+    ps aux | grep -e mailbox-controller -e placement-translator -e cmd/scheduler/main.go
+    user      2898   0.0  0.1 34922264  45188 s004  S     3:22PM   0:01.84 go run cmd/scheduler/main.go -v 2 --root-user shard-main-kcp-admin --root-cluster shard-main-root --sysadm-context shard-main-system:admin --sysadm-user shard-main-shard-admin
+    user      2872   0.0  0.2 34925136  56132 s004  S     3:22PM   0:02.44 go run ./cmd/mailbox-controller --inventory-context=shard-main-root -v=2
+    user      2929   0.0  0.2 34922964  69724 s004  S     3:22PM   0:03.74 go run ./cmd/placement-translator --allclusters-context shard-main-system:admin
+
+    ```
+
+#### 3. Connect your edge pcluster to the kcp-edge platform:
```

Comment:
I think we should stop saying "pcluster", not even as part of "edge pcluster".  The term "pcluster" comes from TMC.  In the context of edge, adding the "p" does not actually add anything.